AES.Web/Controllers/HiringManagerController.cs

```diff
@@ -0,0 +1,393 @@
+﻿using AES.Shared;
+using AES.Shared.Contracts;
+using AES.Web.ApplicationService;
+using AES.Web.Authorization;
+using AES.Web.JobbingService;
+using AES.Web.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace AES.Web.Controllers
+{
+    [AESAuthorize(BadRedirectURL = "/EmployeeLogin", Role = EmployeeRole.HiringManager)]
+    public class HiringManagerController : Controller
+    {
+        // GET: HiringManager
+        public ActionResult DashboardHM()
+        {
+            IApplicationSvc appSvc = new ApplicationSvcClient();
+
+            // ** Get the store ID from the database**********
+            //ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(EmployeeUserManager.GetUser().StoreID);
+            ApplicantInfoContract[] InterviewApplicants = appSvc.GetApplicantsAwaitingInterview(1);
```

Comment:
This shouldn't exist anymore. Not sure how it managed to build. You might need to update your service references.